### PR TITLE
Style Forums and My Forums Page

### PR DESCRIPTION
**Pull request by safchow**
## Trello ticket link

[Developer Task: Implement user feedback for UI/UX of Forums Page](https://trello.com/c/GHAheSjq/135-developer-task-implement-user-feedback-for-ui-ux-of-forums-page)

## Implementation description
* Styled Forums and My Forums Page
* Styled all dialogue options (Archive Forum, Edit Forum)

## Steps to test
1. Verify all dialogue options are working
2. Verify there are no styling inconsistencies
3. Verify correct errors are thrown in error cases

## What should reviewers focus on?
*

### CODE REVIEW

**Review by Eshah7 — CHANGES_REQUESTED** [review]
Looks really nice! Though, the archive forum isn't working. Here are the steps I took: 
1. Loaded branch style-forums-page
2. Went to My Forums
3. Clicked on Archive Forum: 
Screenshot 2023-03-25 at 9 37 35 PM
4. Selected "YES"
5. Waited for the page to refresh
6. The status was still "Active" on the forum. 

Thanks!

**Comment by Eshah7** [comment]
Also, I personally prefer the by: and the name (and the others such as posted date and status) to be next to each other... but that's just a personal preference nbd!

It's a little hard to determine the description!

**Comment by safchow** [comment]
> Looks really nice! Though, the archive forum isn't working. Here are the steps I took:
> 
>     1. Loaded branch style-forums-page
> 
>     2. Went to My Forums
> 
>     3. Clicked on Archive Forum:
> 
> 
> Screenshot 2023-03-25 at 9 37 35 PM
> 4. Selected "YES" 5. Waited for the page to refresh 6. The status was still "Active" on the forum.
> 
> Thanks!

good catch, added a fix

**Comment by safchow** [comment]
> Also, I personally prefer the by: and the name (and the others such as posted date and status) to be next to each other... but that's just a personal preference nbd!
> 
> It's a little hard to determine the description!

I put everything except description on the same line, since descriptions can be paragraphs and it looks funky with multiple lines

**Review by Eshah7 — APPROVED** [review]
lgtm!

File: src/components/forums/MyForums.js
Comment (gabbytang) [inline src/components/forums/MyForums.js:1]: I can't select no tag as an option when editing a forum

**Review by gabbytang — CHANGES_REQUESTED** [review]
Otherwise the forums and my forums pages look great!

File: src/components/forums/MyForums.js
Comment (safchow) [reply src/components/forums/MyForums.js:1]: added fix, thanks for catching this

**Review by gabbytang — APPROVED** [review]

**Pull request merged**